training/finetune_acft_hf_nlp/components/finetune/question_answering/spec.yaml

```diff
@@ -0,0 +1,341 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: question_answering_finetune
+version: 0.0.1
```

Comment:
Please upgrade the version number to match the version number of components in preview registry

Reply:
updated to 0.0.5